build-status.el

```diff
@@ -375,8 +375,7 @@ Signals an error if the response does not contain an HTTP 200 status code."
 ;;;###autoload
 (define-minor-mode build-status-mode
   "Monitor the build status of the buffer's project."
-  :global t
```

Comment:
Can you better explain why this is a problem? 

Currently if you have N buffers open that are all part of the same CI project the mode like will show their status. If this is removed that breaks and one has to enable build-status for *every* buffer. Right?